browser-tests/imposter/config/api-config.yaml

```diff
@@ -319,6 +321,20 @@ resources:
           "page": "prove-identity-another-type-photo-id"
         }
 
+  - method: POST
+    path: /journey/appTriageSmartphone
+    requestHeaders:
+      content-type: "application/json"
+      ipv-session-id: "testIdentifyDeviceIphoneInvalidVersion"
+    queryParams:
+      currentPage: "identify-device"
+    response:
+      statusCode: 200
+      content: |
+        {
+          "page": "prove-identity-another-type-photo-id"
```

Comment:
Is it worth making this (and the one below) a different template? otherwise we don't know for sure that it's matching this event and not `mobileDownloadIphone` above (if I'm reading this correctly?)